.github/workflows/lazy.yml

```diff
@@ -76,3 +76,77 @@ jobs:
               echo "Result: $(expr $FILE_COUNT - $SUCCESS_RESULT_COUNT) / $FILE_COUNT" > counter
           done
           cat counter
+  dneto0_spirv-samples:
+    name: Parse dneto0 spirv-samples
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+      - name: Download shaders
+        run: git clone https://github.com/dneto0/spirv-samples.git
+      - name: Build Naga
+        run: cargo build --release --bin naga
+      - name: Install vulkan-sdk
+        run: |
+           wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo apt-key add -
+           sudo wget -qO /etc/apt/sources.list.d/lunarg-vulkan-1.2.176-focal.list https://packages.lunarg.com/vulkan/1.2.176/lunarg-vulkan-1.2.176-focal.list
```

Comment:
is this necessary? can we just install `vulkan-tools` from the standard package repositories?